index2.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style2.css" />
+    <script
+    src="https://kit.fontawesome.com/f9c0780752.js"
+    crossorigin="anonymous"
```

Comment:
I like that you used Fontawesome, it was easy to use and the sizing was sorted beforehand.